tests/integration/security/util/framework.go

```diff
@@ -38,14 +47,16 @@ const (
 )
 
 type EchoDeployments struct {
-	Namespace     namespace.Instance
-	A, B          echo.Instances
-	Multiversion  echo.Instances
-	Headless      echo.Instances
-	Naked         echo.Instances
-	VM            echo.Instances
-	HeadlessNaked echo.Instances
-	All           echo.Instances
+	Namespace1          namespace.Instance
+	Namespace2          namespace.Instance
+	Namespace3          namespace.Instance
+	A, B, C, D, E, F, X echo.Instances
```

Comment:
Do we really need 7 services? seems excessive... I would not its cheaper to apply a 7 different configs to 1 pod than to have 7 pods with 1 config applied to each